src/validate.ts

```diff
@@ -247,6 +247,15 @@ export function validate(
             ),
           ];
         }
+      } else if (ts.isBindingElement(node)) {
+        if (node.dotDotDotToken && ts.isObjectBindingPattern(node.parent)) {
```

Comment:
`node.parent` is so useful. SWC doesn't have it for memory optimization reasons, it's a bummer.

Reply:
thats too bad, always need to do top down I guess